RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -355,7 +361,7 @@ void equipStatgainIncreasers(boolean[stat] increaseThisStat, boolean alwaysEquip
 		HPlost = my_hp() - simValue("Buffed HP Maximum");
 		MPlost = my_mp() - simValue("Buffed MP Maximum");
 		if(HPlost <= 0 && MPlost <= 0)
-		{	equip(statgainIncreasers[sl],sl);	//causes no loss so it can be equipped right now
+		{	catch equip(statgainIncreasers[sl],sl);	//causes no loss so it can be equipped right now
```

Comment:
probably to not abort autoscend on fail since this script is not essential. though other than mafia side inventory tracking all errors should be fixed now

Reply:
nevermind it doesn't do that. then I don't know, have removed it

Reply:
you'd need to wrap the preceding scope in a try {} block for that to work. The mafia wiki doesn't explain this very well if you've not used this is in other languages.